packages/block-library/src/cover/editor.scss

```diff
@@ -36,6 +36,10 @@
 	}
 }
 
+.wp-block-cover {
+	padding: $grid-unit-20 $grid-unit-40;
```

Comment:
Again, these styles belong at the top of the file. Also, why are we setting the padding in `editor.scss` if we already set it in `style.scss`? And why is the value different here?

Reply:
Thank you for catching that. I'll move this one 👍